bundle/tests/environments_job_and_pipeline_test.go

```diff
@@ -15,8 +14,6 @@ func TestJobAndPipelineDevelopmentWithEnvironment(t *testing.T) {
 	assert.Len(t, b.Config.Resources.Pipelines, 1)
 
 	p := b.Config.Resources.Pipelines["nyc_taxi_pipeline"]
-	l := b.Config.GetLocation("resources.pipelines.nyc_taxi_pipeline")
```

Comment:
Discussed offline with @pietern. We are removing these assertions to keep the unit tests symmetric with the ones in `job_and_pipeline_test.go `